### PR TITLE
Updates to elections-phragmen and some runtime docs. 

### PR DESCRIPTION
**Pull request by kianenigma**
- some leftover doc updates for weights (related to the ongoing doc PR)
- storage -> associated type for elections
- Store the equivalent of `exposure.total` in elections as well. if executed according to the plan, no logic has changed.

needless to say, needs to be taken into account in the next polkadot update.

### CODE REVIEW

File: node/runtime/src/lib.rs [hunk 1]
```diff
@@ -328,6 +328,9 @@ impl collective::Trait<CouncilCollective> for Runtime {
 parameter_types! {
 	pub const CandidacyBond: Balance = 10 * DOLLARS;
 	pub const VotingBond: Balance = 1 * DOLLARS;
+	pub const TermDuration: BlockNumber = 7 * DAYS;
+	pub const DesiredMembers: u32 = 13;
```
Comment (kianenigma) [inline node/runtime/src/lib.rs]: should be replaced maybe with more meaningful numbers.

File: node/runtime/src/lib.rs [hunk 2]
```diff
@@ -82,7 +82,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 189,
```
Comment (gavofyork) [inline node/runtime/src/lib.rs]: definitely a different spec.
```suggestion
	spec_version: 190,
```

File: node/runtime/src/lib.rs [hunk 3]
```diff
@@ -328,6 +328,9 @@ impl collective::Trait<CouncilCollective> for Runtime {
 parameter_types! {
 	pub const CandidacyBond: Balance = 10 * DOLLARS;
 	pub const VotingBond: Balance = 1 * DOLLARS;
+	pub const TermDuration: BlockNumber = 7 * DAYS;
+	pub const DesiredMembers: u32 = 13;
+	pub const DesiredRunnersUp: u32 = 7;
```
Comment (gavofyork) [inline node/runtime/src/lib.rs]:
```suggestion
	pub const DesiredRunnersUp: u32 = 13;
```

File: node/runtime/src/lib.rs [hunk 4]
```diff
@@ -328,6 +328,9 @@ impl collective::Trait<CouncilCollective> for Runtime {
 parameter_types! {
 	pub const CandidacyBond: Balance = 10 * DOLLARS;
 	pub const VotingBond: Balance = 1 * DOLLARS;
+	pub const TermDuration: BlockNumber = 7 * DAYS;
```
Comment (gavofyork) [inline node/runtime/src/lib.rs]:
```suggestion
	pub const TermDuration: BlockNumber = 10 * MINUTES;
```

until we're ready

**Comment by gavofyork** [comment]
needs fixing...

**Comment by kianenigma** [comment]
beast is sick -- have to retry in a short while.

**Comment by gavofyork** [comment]
some legit failures.

**Comment by kianenigma** [comment]
Just ran all the tests locally after the mini-patch. Now it should be all about the CI.

**Pull request merged**